src/api.ts

```diff
@@ -151,6 +151,11 @@ export const githubClientSecret = () =>
 
 export const dataconnectOrigin = () =>
   utils.envOverride("FIREBASE_DATACONNECT_URL", "https://firebasedataconnect.googleapis.com");
+export const dataconnectP4SADomain = () =>
```

Comment:
Not blocking: Is this something that will need to be overridden ever? It seems like all p4sas will have this domain. If it won't be overridden, you could simplify this to just a normal constant in schemaMigration.ts